lib/internal/errors.js

```diff
@@ -440,6 +440,8 @@ E('ERR_OUTOFMEMORY', 'Out of memory');
 E('ERR_OUT_OF_RANGE', 'The "%s" argument is out of range');
 E('ERR_PARSE_HISTORY_DATA', 'Could not parse history data in %s');
 E('ERR_REQUIRE_ESM', 'Must use import to load ES Module: %s');
+E('ERR_SCRIPT_EXECUTION_INTERRUPTED',
+  'Script execution was interrupted by `SIGINT` (Ctrl+C).');
```

Comment:
We probably don't need the part in the parentheses in the error message. The details should be in the docs.